app/src/main/java/me/quadphase/qpdex/Party.java

```diff
@@ -0,0 +1,94 @@
+package me.quadphase.qpdex;
+
+//
+//
+//  Generated by StarUML(tm) Java Add-In
+//
+//  @ Project : QPdex (C) 2015
+//  @ File Name : Party.java
+//  @ Date : 21-Jul-15
+//  @ Author : Nicole
+//
+//
+
+
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * These <code>Pokemon</code> are in your party. They each have a particular set of moves.
+ */
+public class Party {
+
+    /**
+     * Max: 6
+     */
+    private List<Pokemon> pokemon;
+
+    /**
+     * 1 move set per Pokemon in the <code>pokemon</code> list
+     */
+    private List<MoveSet> moveSets;
+
+    /**
+     * Constructor to make an empty party
+     */
+    public Party() {
```

Comment:
I don't think it would change much in the database. It would just be a matter of putting a **userID** column in the **party** and **party_moveSet** tables.

That being said, how useful would it be to have?